utils.test.js

```diff
@@ -2220,8 +2220,8 @@ test("parsePkgLock v3", async () => {
     projectName: "cdxgen",
   });
   deps = parsedList.pkgList;
-  expect(deps.length).toEqual(842);
```

Comment:
I am not sure about using the full express since it is bringing 19 more packages. Any issues with connect?

Reply:
Yes, as documented in #1075 connect hasn't seen a single commit in ~4 years. I'd consider this critical to replace with something maintained and current. 

Additionally, I have a local branch shortly ready to share based on express that can generate an OpenAPI Specification from jsdocs, so that we can avoid using md to document API Specifications

Reply:
Express appears to be using connect internally. OpenAPI sounds interesting, too.